index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
```

Comment:
I could have used header tag to make it look more appealing

Reply:
I believe the reason is not stated correctly. Can you please rectify it?

As per the good HTML semantics, we have to follow the following structure:

```
<header>
  Code for header
</header>

<main>
   <section id="..." class="..." >
     ...
   </section>
     
   <section id="..." class="...">
      ...
   </section>
</main>

<footer>
   Code for the footer
<footer>
```